### PR TITLE
[Linux][Manager][vcpkg] Compile Linux vcpkg manager in manifest mode.

### PR DESCRIPTION
**Pull request by talregev**
Compile Linux vcpkg manager in manifest mode.
tested on wsl2 ubuntu 22.04.

### CODE REVIEW

**Comment by talregev** [comment]
image

File: 3rdParty/vcpkg_ports/configs/manager/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "boinc-msbuild-x64",
```
Comment (AenBleidd) [inline 3rdParty/vcpkg_ports/configs/manager/vcpkg.json:2]: Is this is for linux, then please change it to `boinc-manager` and put into 'linux' subdirectory

**Review by AenBleidd — CHANGES_REQUESTED** [review]
Please update https://github.com/BOINC/boinc/blob/master/set-client-version.py script to include your new file

**Comment by talregev** [comment]
We should remove all versions from vcpkg.json. there is not meaning / not effecting boinc versioning. 
I will test it in other PR.

File: 3rdParty/vcpkg_ports/configs/manager/vcpkg.json (hunk 1 above)
Comment (talregev) [reply 3rdParty/vcpkg_ports/configs/manager/vcpkg.json]: done.

**Review by AenBleidd — APPROVED** [review]

**Comment by AenBleidd** [comment]
Looks good to me.
@talregev, thank you for the contribution.

**Pull request merged**